python-lib/rest_api_recipe_session.py

```diff
@@ -52,6 +52,7 @@ def process_dataframe(self, input_parameters_dataframe, is_raw_output):
     def retrieve_next_page(self, is_raw_output):
         page_rows = []
         base_row = copy.deepcopy(self.initial_parameter_columns)
+        logger.info("retrieve_next_page: Calling next page")
```

Comment:
maybe log the token?

Reply:
:+1 Added logs in has_next_page where the logic takes place.